package.json

```diff
@@ -24,6 +24,10 @@
     "scripts/*",
     "shared/*"
   ],
+  "nohoist": [
+    "examples/svelte-kit/vite/**",
+    "examples/vue-with-typescript/vite/**"
+  ],
```

Comment:
Don't miss this root level change! It seems important. I don't think it broke anything, but since monorepos scare me, maybe y'all should double check 👀